src/less/less/ds6/variables.less

```diff
@@ -288,6 +288,12 @@
 @ds6-icon-red-laser-height: 18px;
 @ds6-icon-window-width: 28px;
 @ds6-icon-window-height: 25px;
+@ds6-icon-confirmation-foreground-width: 14px;
+@ds6-icon-confirmation-foreground-height: 14px;
+@ds6-icon-attention-foreground-width: 14px;
+@ds6-icon-attention-foreground-height: 14px;
+@ds6-icon-information-foreground-width: 14px;
```

Comment:
I believe the background dimensions should always be the same as the foreground dimensions, so we haven't had to specify "foreground" or "background" in the variable name.

Reply:
got it, I'll update that; the SVGs had a `height` and `width` of 14 each, so I figured they needed to be 14 rather than 24. Either way, I can change it